src/sentry/lang/javascript/processor.py

```diff
@@ -220,9 +220,12 @@ def fetch_file(url, project=None, release=None, allow_scraping=True):
 
     Attempts to fetch from the cache.
     """
-    cache_key = 'source:cache:v2:%s' % (
-        md5(url).hexdigest(),
-    )
+    if url is None:
```

Comment:
should it really get here with url being None? seems like this should be taken care of somewhere else